.github/filters.yml

```diff
@@ -0,0 +1,24 @@
+shell:
+  - 'e2e/**'
+  - '.github/**'
```

Comment:
It looks like technically this doesn't need to be here, because every workflow also checks the workflows filter
```suggestion
```